setup-guide.md

```diff
@@ -64,5 +64,412 @@ $git config --global core.autocrlf true
 > npx eslint . --fix
 > ```
 
+## Migration
```

Comment:
let's move this migration section to another file and leave a link to it under `## Move to new project` section